src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -210,8 +195,6 @@ internal static bool HasIllegalCharacters(string path)
                 switch (c)
                 {
                     case '"':
```

Comment:
Doh. Thanks- trying to code when sick- I did have a test and it is failing. :mask: I'm going to move the one remaining case up to the other if block.